src/testing/systest/workload/src/main/java/Main.java

```diff
@@ -17,12 +19,22 @@ public static void main(String[] args) throws Exception {
           "REPLICAS must list at least one address (comma-separated)");
     }
 
+    int workloadCount = Integer.parseUnsignedInt(env.getOrDefault("WORKLOAD_COUNT", "5"));
+    if (replicaAddresses.length == 0) {
+      throw new IllegalArgumentException(
+          "REPLICAS must list at least one address (comma-separated)");
+    }
+
     Random random = new SecureRandom();
     byte[] clusterID = UInt128.asBytes(Long.parseLong(env.getOrDefault("CLUSTER", "1")));
 
     try (var client = new Client(clusterID, replicaAddresses)) {
-      System.err.println("starting workload");
-      new Workload(random, client).run();
+      try (var scope = new StructuredTaskScope.ShutdownOnFailure()) {
+        System.err.println("starting %d workload(s)".formatted(workloadCount));
+        IntStream.rangeClosed(1, workloadCount)
+          .forEach(ledger -> scope.fork(new Workload(random, client, ledger)));
+        scope.join();
```

Comment:
Do you need join? I'd imagine STS is AutoClosable and would work with try with resources?